bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/LimitsListener.java

```diff
@@ -38,6 +58,12 @@ public void propertyChange(PropertyChangeEvent evt) {
 		MarkerManager mm = v.getSchedule().getMarkerManager();
 		mm.clearMarkers(this, v);
 
+		final TimeZone zone = v.getSchedule().getSite().timezone();
+
+		final Site site = v.getSchedule().getSite();
+
+		final SortedSet<Block> blocks = v.getSchedule().getBlocks();
```

Comment:
I don't think we use the `blocks` anywhere now, correct?  For that matter, maybe `site` isn't used anywhere now either?